keyboards/tominabox1/le_chiffre/keymaps/via/keymap.c

```diff
@@ -144,13 +144,14 @@ void render_mod_status(uint8_t modifiers) {
     oled_write_ln_P(PSTR("GUI"), (modifiers & MOD_MASK_GUI));
 }
 
-bool oled_task_user(void) {
+void oled_task_keymap(void) {
```

Comment:
Pretty sure this is supposed to be:
```suggestion
bool oled_task_user(void) {
```